### PR TITLE
[develop] Lucero - Adds test to invite member to a board.

### LINKED ISSUE #60: Smoke Tests Boards

### PR DESCRIPTION
**Pull request by luqp**

### CODE REVIEW

File: src/test/java/org/fundacionjala/trello/pages/menus/Header.java [hunk 1]
```diff
@@ -66,4 +74,19 @@ public MenuProfile getMenuProfile() {
...
+        action.click(notificationBtn);
+        return this;
+    }
+
+    public BoardPage selectBoardNotification(final String type, final String boardName) {
+        String notificationPath = String.format(NOTIFICATION_PATH, type, boardName);
+        By notifyLocator = By.xpath(notificationPath);
+        action.waitForElementLocated(notifyLocator);
+        WebElement notification = driver.findElement(notifyLocator);
+        notification.click();
```
Comment (nanrehd) [inline src/test/java/org/fundacionjala/trello/pages/menus/Header.java]: use the action class to perform this click

File: src/test/resources/features/boards/Invite_Members.feature [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    And I navigate to notifications
+    * I open notification: added to "Juan Party" board
+    * "Juan Party" board page should be visible
+    And I navigate to boards home page
+    * I navigate to "member" section
+    * I select "Juan Party" board
+    * "Juan Party" board page should be visible
+    And I navigate to boards menu from header
+    * I open the "Juan Party" board
+    * "Juan Party" board page should be visible
```
Comment (nanrehd) [inline src/test/resources/features/boards/Invite_Members.feature]: add a new line at the end of file

File: src/test/java/org/fundacionjala/trello/pages/menus/Header.java (hunk 1 above)
Comment (luqp) [reply src/test/java/org/fundacionjala/trello/pages/menus/Header.java]: Done

File: src/test/resources/features/boards/Invite_Members.feature (hunk 2 above)
Comment (luqp) [reply src/test/resources/features/boards/Invite_Members.feature]: Done

**Review by Snip-77 — APPROVED** [review]

**Pull request merged**